### PR TITLE
(✨) : Added functionality to delete children and activities

### PR DESCRIPTION
**Pull request by d-azmanov**
- Functionality to allow parents to delete children from their profile
     - Deletes the child as well as all of their associated data (activities) from the database.
- Functionality to allow parents to delete existing activities for a child

### CODE REVIEW

**Review by priyo5 — DISMISSED** [review]

File: libs/add-child/feature/src/lib/add-child.component.ts [hunk 1]
```diff
@@ -45,6 +45,8 @@ export class AddChildComponent implements OnInit
   //Function to retrieve the child's details
   async getChildValues(val: any)
   {
+    console.log("Children: ", this.allChildren);
```
Comment (ricecakesreece) [inline libs/add-child/feature/src/lib/add-child.component.ts:48]: console.log

File: libs/children-dashboard/feature/src/lib/children-dashboard.component.scss [hunk 2]
```diff
@@ -76,15 +76,27 @@ ion-card {
...
+    cursor: pointer;
+    top: 11px;
+    margin-left: 40px;
+    align-items: center;
+    position: relative;
+    display: inline-block;
+    align-items: center;
+    color: var(--ion-color-danger);
+}
+
```
Comment (ricecakesreece) [inline libs/children-dashboard/feature/src/lib/children-dashboard.component.scss:98]: spaces

File: libs/children-dashboard/feature/src/lib/children-dashboard.component.scss [hunk 3]
```diff
@@ -166,11 +181,19 @@ ion-card {
...
     }
+
+    .remove-child{
+        cursor: pointer;
+        font-size: 20px;
+        margin-bottom: 0;
+        margin-top:0;
+        top: 4px;
+    }
     .action {
```
Comment (ricecakesreece) [inline libs/children-dashboard/feature/src/lib/children-dashboard.component.scss:197]: no space and previous comment left in

File: libs/children-dashboard/feature/src/lib/children-dashboard.component.scss [hunk 4]
```diff
@@ -189,45 +212,43 @@ ion-card {
...
-    .action-tag {
-        height: 45px;
-        font-size: 18px;
-    }
-    .action-icon {
-        height: 96px;
-        width: 96px;
-    }
-}
+// @media (max-width: 810px) {
```
Comment (ricecakesreece) [inline libs/children-dashboard/feature/src/lib/children-dashboard.component.scss:215]: is all of this code meant to be commented out?

**Review by priyo5 — APPROVED** [review]

**Review by ricecakesreece — APPROVED** [review]

**Pull request merged**